src/components/dev-dot-content/mdx-components/mdx-code-blocks/index.tsx

```diff
@@ -220,20 +220,22 @@ function childrenOfFirstChild(children: ReactNode): ReactNode {
 		'props' in firstElement
 	) {
 		return firstElement.props.children
+	} else {
+		return null
 	}
-
-	throw new Error('First element of children is not an object with props')
 }
 
-function propsOfFirstChild(children: ReactNode): Record<string, unknown> {
+function getPropsOfFirstChild(
```

Comment:
Small point but what do you think about calling this `getFirstChildProps`?

Reply:
I dig it, thanks for the suggestion! Renamed and pushing the commit now 👍